index.html

```diff
@@ -35,39 +35,43 @@ <h1 class="welcome">Welcome</h1>
 					</div>
 				</div>
 
-			</div>
+			</nav>
 
 			<!-- intro -->
-			<div class="intro">
-				<div class="primary">
-					<p class="primary-text"
-					   id="primary">Hey there. I’m Jared 
-					   <span id="career">I’m a software
-						   developer</span></p>	
+			<header class="intro">
+				<div>
+					<h1 class="primary-text"
+					    id="primary">Hey there. I’m Jared 
+				</div>
+				<div>
+					<h1 class="primary-text"id="career">I’m a software
```

Comment:
- [ OPTIONAL ] I recommend keeping the heading color the same as the Figma design since it is one of the requirements to stick with the Figma design and font and colors so kindly change it to the fimga color.

- Your design
![image](https://user-images.githubusercontent.com/58553711/198179459-31d2a50f-eae7-47ab-b09b-517c64cd5633.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/198179525-2030822a-507d-4b2a-b8d0-4eea45322428.png)